### ISSUE TITLE
"Search in multiple nodes" instead opens "Replace in multiple nodes"

### ISSUE THREAD

**Issue by smeszner-etes** [giuspen/cherrytree#2468]
**Version, Operative system**
1.1.0, Ubuntu 22.04 LTS

**Describe the bug**
Its not possible to open the dialog to search in multipe nodes. It instead opens the dialog to replace in multiple nodes.
Keyboard Shortcuts also doesnt work. Because of this bug, I lost information in all my notes.

**Screenshots**
![search_bug](https://github.com/giuspen/cherrytree/assets/60345194/5e44df31-56fa-4ef2-899a-5c0a4e622ee9)

**Comment by giuspen** [comment]
What kind of package are you using? Deb from giuspen PPA / snap / flatpak / appimage?
You can download the older version AppImage from the GitHub release page

**Comment by smeszner-etes** [comment]
I use the official repo:
```
/etc/apt/sources.list.d$ cat giuspen-ubuntu-ppa-jammy.list 
deb https://ppa.launchpadcontent.net/giuspen/ppa/ubuntu/ jammy main 
```

![grafik](https://github.com/giuspen/cherrytree/assets/60345194/1df1b9f8-11da-4d45-88fb-2dd19f06ac1f)

**Comment by giuspen** [comment]
Could you export and share your config.cfg? File--Preferences--Export
Could you also download and try the AppImage file and report if behaves the same?

**Comment by giuspen** [comment]
Also please download the AppImage for 1.0.4 and report how that behaves

**Comment by smeszner-etes** [comment]
[config_2024.03.26_19.42.46.txt](https://github.com/giuspen/cherrytree/files/14763469/config_2024.03.26_19.42.46.txt)

**Comment by smeszner-etes** [comment]
With AppImage 1.0.4 it works:

![AppImage_works](https://github.com/giuspen/cherrytree/assets/60345194/5850b3d8-840c-4796-b5d1-4f958f9ab2c0)

**Comment by giuspen** [comment]
Can you try AppImage v1.1.0 too please?

**Comment by giuspen** [comment]
May it be that it starts to behave in this weird way not from the moment you start cherrytree but only after some time that you are using it?

**Comment by smeszner-etes** [comment]
Works fine with AppImage 1.1.0

> May it be that it starts to behave in this weird way not from the moment you start cherrytree but only after some time that you are using it?
 
Yes, youre right, restarted my ppa cherrytree and for the moment it works. I wonder when it starts misbeheaving.
I have to go, my end of work is long overdue.

**Comment by giuspen** [comment]
Thanks @smeszner-etes it starts to make sense now, I'll investigate and get back

**Comment by BileDemon** [comment]
I can confirm this bug in the 1.1.0 Test build AppImage.

I can replicate it like this:
When you open CherryTree, you can select "Search in multiple nodes" as often as you want, it will always show the proper dialog.
BUT once you select "Replace in multiple nodes", it will always show this replace-dialog for "Search in multiple nodes" too. So this triggers the bug for me. I have to restart CherryTree to get the search dialog to open again.

**Comment by giuspen** [comment]
This should be now fixed. If you could try the test build from https://www.giuspen.net/cherrytree/#testing (called 1.1.0+10) and report if that works as expected or still has issues it would be very useful. The plan is to release v1.1.1 tomorrow the 27th

**Comment by giuspen** [comment]
Fixed in v1.1.1
